### ISSUE TITLE
Actions [f1] Sole action(Identity)

### ISSUE THREAD

**Issue by nfedyashev** [emacs-jp/helm-c-yasnippet#23]
I'm currently experiencing a bit strange behavior of helm-yas so any tips are greatly appreciated.

`yas/insert-snippet` 
*type something*
press `ctrl` `z` and there is just one option displayed there

> Select action:
> Actions
> [f1] Sole action(Identity)

What could be the reason for it? It happens only in the specific major-mode for me - `typescript-mode`. All the other modes and snippets have all the actions listed.

**Comment by syohex** [comment]
I'm not sure. It looks same behavior other than typescript-mode when I use `yas-insert-snippet`. How about using `helm-yas-complete` instead of `yas-insert-snippet` ?

**Comment by nfedyashev** [comment]
`helm-yas-complete` shows "No JavaScript AST available" error.
It worked fine before I attempted to reinstall Emacs :) Sorry for that confusion, I'll try again.

**Comment by syohex** [comment]
Ah, it is this package issue. I'll fix it later.

**Comment by syohex** [comment]
@nfedyashev I have merged #24. Could you try `helm-yas-complete` with latest version ?

**Comment by nfedyashev** [comment]
@syohex wow! It is working again after package upgrade to the latest version.
You're the boss!

**Comment by nfedyashev** [comment]
@syohex hi again

Just a minor comment. I decided to downgrade my emacs back to 26.1 to avoid possible conflicts with other plugins.

Then I noticed the warning 
>the function ignore-error is not known to be defined

I've tried to change it to `ignore-errors` because there seems to be plenty of `ignore-errors` but only a single `ignore-error` search result in my .emacs.d folder. After that change I recompiled all plugins, and helm/yas works fine again. No warnings, and all actions are listed.

I no very little elisp but at first sight it looked like `ignore-error` is available in most recent emacs versions but not in 26.1(which is still pretty popular). That's just my guess. Hope that change didn't break it for other users.
